index.js

```diff
@@ -1,5 +1,4 @@
 import CircularProgress from './src/CircularProgress';
 import AnimatedCircularProgress from './src/AnimatedCircularProgress';
 
-exports.CircularProgress = CircularProgress;
-exports.AnimatedCircularProgress = AnimatedCircularProgress;
+export { CirculartProgress, AnimatedCircularProgress }
```

Comment:
There seems to be a spelling error here `CirculartProgress` -> `CircularProgress`